src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -105,25 +98,23 @@ void MulticopterLandDetector::_update_topics()
 
 void MulticopterLandDetector::_update_params()
 {
-	param_get(_paramHandle.maxClimbRate, &_params.maxClimbRate);
-	param_get(_paramHandle.maxVelocity, &_params.maxVelocity);
-	param_get(_paramHandle.maxRotation, &_params.maxRotation_rad_s);
-	_params.maxRotation_rad_s = math::radians(_params.maxRotation_rad_s);
+	parameter_update_s param_update;
+
+	if (_param_update_sub.update(&param_update)) {
```

Comment:
You don't need to check for this, or call `_update_params()`. It is handled and called from the base class.
You can also keep `_param_update_sub` private.

Reply:
Great, thanks! That simplifies things. I pushed up that fix.